lib/get-acey-deucey.js

```diff
@@ -1,18 +1,31 @@
 "use strict";
 
+const _ = require("lodash");
+
 function getAceyDeucey(diceRoll) {
     const aceyDeucey = {};
-    diceRoll.forEach(function(roll) {
-        if (roll === 1 && !aceyDeucey.hasOne) {
-            aceyDeucey.hasOne = true; 
-        } else if (roll === 2 && !aceyDeucey.hasTwo) {
-            aceyDeucey.hasTwo = true;
-        } else if (!aceyDeucey.doublesVal) {
-            aceyDeucey.doublesVal = roll;
-        } else {
-            aceyDeucey.isAceyDeucey = false;
+    if (diceRoll.length !== 3) {
+        if (diceRoll.length === 2 && _.includes(diceRoll, 1) && _.includes(diceRoll, 2)) {
+            const err = new Error("This acey deucey roll does not have a doubles value");
+            err.diceRoll = diceRoll;
+            throw err;
         }
-    });
+        aceyDeucey.isAceyDeucey = false;
+        return aceyDeucey;
+    }
+    
+    aceyDeucey.hasOne = _.includes(diceRoll, 1);
+    aceyDeucey.hasTwo = _.includes(diceRoll, 2);
+    if (aceyDeucey.hasOne && aceyDeucey.hasTwo) {
+        let doubles = diceRoll.slice(0);
+        doubles.splice(_.indexOf(doubles, 1), 1);
+        doubles.splice(_.indexOf(doubles, 2), 1);
+        aceyDeucey.doublesVal = doubles[0]; 
```

Comment:
Wouldn't that take out all 1s and 2s? It's possible to have 1 or 2 as the doubles val.